project_singlefamilydetached/singlefamilydetached_with_upgrades.yml

```diff
@@ -0,0 +1,1165 @@
+#this is intended to be a nearly complete YAML implementation of the 2014 Building America House Simulation Protocols
+
+stock_type: residential
+buildstock_directory: ../../OpenStudio-BuildStock  # Relative to this file or absolute
+project_directory: project_singlefamilydetached
+output_directory: /scratch/epresent/outputs/project_singlefamilydetached #location on eagle where outputs should go - must not currently exist
```

Comment:
```suggestion
output_directory: /scratch/$USER/outputs/project_singlefamilydetached #location on eagle where outputs should go - must not currently exist
```

Reply:
I'm not so sure this would work... @nmerket ?

Reply:
```suggestion
output_directory: /scratch/<username>/outputs/project_singlefamilydetached #location on eagle where outputs should go - must not currently exist
```

Reply:
Just trying to keep real usernames out of github. Is that valuable? I don't really care how it gets done. 🤷‍♂